css/chat.css

```diff
@@ -18,6 +18,12 @@ html.no-supports-container.native body,
   height: calc(100vh - 44px);
 }
 
+@media screen and (max-width: 640px) {
+  .supports-container .fl-with-bottom-menu .fl-widget-instance[data-name="Chat"] {
```

Comment:
Let's change `[data-name="Chat"]` to use `data-package` instead as it's less likely to change over time

Reply:
Should I change all the occurrences(7 places) in chat.css file to [data-package="com.fliplet.chat"]

Reply:
@NikitaJadhavXoriant Yes please

Reply:
Changes done